### ISSUE TITLE
Failed to convert some files

### ISSUE THREAD

**Issue by Soeonye** [YukihoAA/waifu2x_snowshell#31]
**Describe the bug**
When I try to convert,

> Failed to convert some files. Check "error.log"

and there is no information in error.log.

**To Reproduce**
1. Use 'waifu2x-converter-cpp' or 'waifu2x-caffe'
2. Click on 'Prefer GPU'
3. Convert

**Expected behavior**
Regular run of the application.

**Screenshots**
https://imgur.com/NFaOAsT
https://imgur.com/gFYJS1u

**Hardware Info (please complete the following information):**
 - CPU: Intel i5-6500
 - RAM: 8GB
 - GPU: NVIDIA GTX 750 Ti
 - OS: Windows 10
 - Converter: waifu2x-converter-cpp, waifu2x-caffe

**Additional context**
If I use 'Vulkan' or use 'Force CPU only', no error occurs. However, it is inconvenient because the 'Force CPU only' settings are not saved, and I want to know why this is happening.
As a result of checking, the GPU memory is not insufficient at all.

**Comment by YukihoAA** [comment]
1. did you downloaded fullpack from release page?
2. does your graphic driver is lastest?

I checked cpu option does not saving, it will fixed next version.

**Comment by Soeonye** [comment]
Yes, I used both the light version and the full version, and confirmed that both had the same problem.
I also used both the latest version of the graphics driver and the new version updated today, but I found the same problem.

**Comment by YukihoAA** [comment]
try new release!

**Comment by Soeonye** [comment]
Actually, it still throws an error using the GPU.
However, the error.log comes out normally (though still no reason for the error), and the 'Force CPU only' setting is now saved.
Thanks for your attention to my problem!
I paste the contents of error.log here in case it might be helpful to you:

[Converter] 
Current: E:\waifu2x-snowshell_v2.6.1_with_fullpack\waifu2x-converter\waifu2x-converter-cpp.exe
Model: 
Option: 
WorkDir: E:\waifu2x-snowshell_v2.6.1_with_fullpack\waifu2x-converter

[System] 
Cuda: OK
Vulkan: OK
waifu2x-converter-cpp: OK
waifu2x-caffe: OK
waifu2x-vulkan: OK
waifu2x-CUGan: OK
waifu2x-ESRGan: OK

[proc]
   0: NVIDIA GeForce GTX 750 Ti                      (CUDA      ): num_core=5

   1: NVIDIA GeForce GTX 750 Ti                      (NVIDIA-GPU): num_core=5

   2: Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz        (FMA       ): num_core=4

Error: E:\waifu2x-snowshell_v2.6.1_with_fullpack\001.jpg
